src/chapters/methods/exercises.rst

```diff
@@ -0,0 +1,188 @@
+Exercises: Methods
+=====================
+
+To become good at solving problems with code you need to be able to break large
+problems into small ones. Usually, these smaller problems will take the form of
+methods that are used to solve the larger problem. Therefore, to become good
+at solving problems you need to become good at writing methods. And to master
+methods, you need to write a *lot* of them.
+
+These exercises ask you to write lots of methods, starting with 
+small one and end with a larger, more complicated one.
+
+At the end, you will be able to create strings of shapes, like this nifty
+diamond:
+
+::
+
+       #
+      ###
+     #####
+    #######
+   #########
+   #########
+    #######
+     #####
+      ###
+       #
+
+.. admonition:: Note
+
+   As you build your methods, remember to keep the output in mind.  
+   Do you want a value returned or are do you want the method to print directly to the console when invoked?
+   How many things are being repeated?
+
+There is no starter code for these exercises, so create a new C# project
+at `repl.it <https://repl.it/login>`__ to get started.
+
+.. admonition:: Tip
+
+   As you create these methods, keep the shape of your output in mind.  
+   Remember the differences between ``Console.WriteLine`` and ``Console.Write`` .  
+
+
+Rectangles
+----------
+
+#. Write a method ``MakeLine(size)`` that returns a line with exactly ``size``
+   hashes.
+
+   .. sourcecode:: csharp
+
+      MakeLine(5);
+
+   **Console Output**
+
+   ::
+
+      #####
```

Comment:
we should provide the signature for at least the first of these methods to get students into the groove. also though, if we're wanting these methods to return these shapes, then the "Console Output" will not be anything unless Console.WriteLine(MakeLine(5)) is called, right?

Maybe we should ask the students instead to write methods that return void but print the shapes instead? having the shapes be returned would require more practice with return types though

Reply:
does this mean we should give them the first method?  or the method template?  

i used void methods.